frontend/src/lib/components/PropertyFilters/components/PropertyFilterBetween.tsx

```diff
@@ -0,0 +1,41 @@
+import { useActions, useValues } from 'kea'
+
+import { LemonInput } from 'lib/lemon-ui/LemonInput/LemonInput'
+
+import { PropertyFilterValue } from '~/types'
+
+import { propertyFilterBetweenLogic } from './propertyFilterBetweenLogic'
+
+export interface PropertyFilterBetweenProps {
+    value: PropertyFilterValue
+    onSet: (newValue: PropertyFilterValue) => void
+    size?: 'xsmall' | 'small' | 'medium'
+}
+
+export function PropertyFilterBetween({ value, onSet, size }: PropertyFilterBetweenProps): JSX.Element {
+    const logic = propertyFilterBetweenLogic({ value, onSet })
+    const { localMin, localMax } = useValues(logic)
+    const { setLocalMin, setLocalMax } = useActions(logic)
+
+    return (
+        <div className="flex items-center gap-2">
+            <LemonInput
+                type="number"
+                value={localMin ?? undefined}
+                data-attr="prop-val"
```

Comment:
For accessibility, probably a good idea to set the `aria-label`. Also, both inputs have the same `data-attr`. I think it's good to have these be unique so they're easier to select if we write Playwright tests, etc.

```suggestion
                data-attr="prop-val-min"
                aria-label="Minimum value"
```